core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -412,6 +412,9 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     StrengthBonusVsCityStates("+30% Strength when fighting City-State units and cities", UniqueTarget.Global),
     StrengthForAdjacentEnemies("[relativeAmount]% Strength for enemy [combatantFilter] units in adjacent [tileFilter] tiles", UniqueTarget.Unit),
     StrengthWhenStacked("[relativeAmount]% Strength when stacked with [mapUnitFilter]", UniqueTarget.Unit),  // candidate for conditional!
+    @Deprecated("as of 4.0.15", ReplaceWith("[15]% Strength bonus for [Military] units in a [2] tile radius"))
+    BonusForUnitsInRadius("Bonus for units in 2 tile radius 15%", UniqueTarget.Unit),
+    GreatGeneralAura("[relativeAmount]% Strength bonus for [mapUnitFilter] units in a [amount] tile radius", UniqueTarget.Unit),
```

Comment:
I personally prefer `[relativeAmount]% Strength for [mapUnitFilter] units in a [amount] tiles radius`, as the `+` in the relative amount should indicate the bonus. If this were to be negative (for some reason, don't ask me why you'd want that), a `[-20]% bonus` sounds a bit weird :upside_down_face:

Reply:
Took me a second to grok - you suggest removing the word "bonus". OK, accepted. Yes, also matches the rest of the game better.

I'll wait for more feedback - is the maximized flexibility I went for overkill and JR's approach may please more? If so, point moot.